### PR TITLE
fix(flat-server-api): fix email language error

### PR DESCRIPTION
**Pull request by syt-honey**
I wonder if phone numbers also need to be differentiated by language? @Leooeloel

### CODE REVIEW

**Comment by hyrious** [comment]
> I wonder if phone numbers also need to be differentiated by language?

I guess no. The server has set different template for different country codes https://github.com/netless-io/flat-server/blob/main/config/test.yaml#L113 (although it can only tell the prefix).

**Comment by syt-honey** [comment]
> although it can only tell the prefix

OK 👌

**Comment by syt-honey** [comment]
> > I wonder if phone numbers also need to be differentiated by language?
> 
> I guess no. The server has set different template for different country codes https://github.com/netless-io/flat-server/blob/main/config/test.yaml#L113 (although it can only tell the prefix).

OK, you are right

File: packages/flat-server-api/src/login.ts [hunk 1]
```diff
@@ -255,12 +263,18 @@ export async function resetPwdWithEmail(
     });
 }
 
-export async function resetEmailSendCode(email: string): Promise<EmailSendCodeResult> {
+export async function resetEmailSendCode(
+    email: string,
+    language?: string,
+): Promise<EmailSendCodeResult> {
+    const payload: EmailSendCodePayload = { email };
```
Comment (hyrious) [inline packages/flat-server-api/src/login.ts]: Why not just&hellip;

```ts
const payload: EmailSendCodePayload = { email, language };
```

&hellip;?

Comment (syt-honey) [reply packages/flat-server-api/src/login.ts]: You're right. I'll change it now

**Review by hyrious — APPROVED** [review]

**Pull request merged**